websites/P/Perplexity/presence.ts

```diff
@@ -0,0 +1,86 @@
+const presence = new Presence({
+		clientId: "893871116945350726",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	threadExportRegex = /([a-f\d]{8}(-[a-f\d]{4}){3}-[a-f\d]{12})/i;
+let recentSearchQuery: string = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "lg",
```

Comment:
We use imgur links for all non element large & small image keys
```suggestion
			largeImageKey: "https://i.imgur.com/gdCcljx.png",
```